internal/handlers/volume_handler.go

```
@@ -0,0 +1,210 @@
+	})
+}
+
+func (h *VolumeHandler) DeleteVolume(c *gin.Context) {
+	volumeID := c.Param("id")
+
+	var req struct {
+		Force bool `json:"force"`
+	}
+
+	// Check for force parameter in query or body
+	c.ShouldBindJSON(&req)
```

golangci-lint (1.64.8)

Error return value of `c.ShouldBindJSON` is not checked

(errcheck)